topostats/tracing/dnatracing.py

```diff
@@ -493,69 +488,88 @@ def get_splined_traces(self):
                 if not self.splining_success:
```

Comment:
Further I think the naming convention could be clearer.

It requires a fraction more thought to think what the result of

```
if not splinng_success:
```

...is (for me at least!).

It would be clearer to have...

```
splining_failure = False
```

...defined at the start of the method and then within the `except ValueError:`...

```
except ValueError:
    ...
    splining_failure True
...

if splining_failure:
    continue
```

That said I'm not sure you even need this `if not self.splining_success:` or `if splining_failure:` at all since the errors are captured by virtue of the `try: ... except: ...` statements and since there is nothing beyond `continue` in the `if:` statement then its kind of redundant.